app/controllers/posts_controller.rb

```diff
@@ -8,6 +8,7 @@ def show
     @post = Post.find(params[:id])
     @comments = @post.comments
     @likes = @post.likes
+    @user_s = User.find(@post.user_id)
   end
 
```

Comment:
- [ ] Let's ensure that we solve the `n + 1` query problem by including the comments when fetching a user's posts in the `index` action. This way, we only get to query the database once when fetching posts because we included the comments on that query. But when you do not include the comments, the database is queried based on the number of comments the user has which results in a `n + 1` query problem. You can check this [article](https://dev.to/junko911/rails-n-1-queries-and-eager-loading-10eh#:~:text=What%20is%20the%20N%2B1,the%20result%20of%20that%20data.) for more explanation on `n + 1` query problem in rails and how to solve it. I couldn't comment on the index action because it wasn't modified in this PR. Hint below;


```ruby
 def index
    @user = User.find(params[:user_id])
    @posts = @user.posts.includes(:comments)
  end
```